documentation/asciidoc/computers/raspberry-pi/frequency-management.adoc

```diff
@@ -59,14 +59,16 @@ Thanks to built-in throttling, heatsinks are not necessary to prevent overheatin
 
 === Fan cases
 
+For the Raspberry Pi 4, you have the option of adding the https://www.raspberrypi.com/products/raspberry-pi-4-case-fan/[Raspberry Pi 4 Case Fan] to the lid of the Raspberry Pi 4 case.
```

Comment:
Can we get an introduction sentence, instead of jumping straight into Raspberry Pi 4 info? Maybe something like: "To ensure the best performance for your Raspberry Pi, use an active cooling solution such as a fan."

Then, we could either break this up into "Raspberry Pi 4" and "Raspberry Pi 5" sections (slightly preferred), or perhaps provide info about each model in an unordered list. Either way, we should probably restructure this sentence introduction to better match the Raspberry Pi 5 equivalent:

```suggestion
For Raspberry Pi 4, add the https://www.raspberrypi.com/products/raspberry-pi-4-case-fan/[Raspberry Pi 4 Case Fan] to the lid of the Raspberry Pi 4 case.
```

And I'd probably modify the line below to something like "For Raspberry Pi 5, use one of the official fan options:"

Reply:
Building on the hysteresis comment below, perhaps add a sentence like this to the introduction:

"Raspberry Pi firmware manages fan speeds for all official fans."

That way, it's clear that the Raspberry Pi 4 fan doesn't just blow at 100% all of the time :-)